src/locales/en.json

```diff
@@ -231,6 +231,43 @@
     "whyWeDoItHeader": "Why we do it",
-    "whyWeDoItText": "We have a moral obligation to preserve nature and its features, because protecting the environment protects humanity from toxic substances and our quality of food. We believe that the new generation in MENA should be supported with the right tools to believe and act for the benefit of humanitarian and social issues, on the local and global scale. Given the right tools, it is not impossible to raise a generation of activists."
+    "whyWeDoItText": "We have a moral obligation to preserve nature and its features, because protecting the environment protects humanity from toxic substances and our quality of food. We believe that the new generation in MENA should be supported with the right tools to believe and act for the benefit of humanitarian and social issues, on the local and global scale. Given the right tools, it is not impossible to raise a generation of activists.",
+    "teamInfo": {
+      "louis": {
+        "name": "Louis Li",
+        "bio": "Louis is the coding instructor in Re:Coded Turkey. With a lot of exciting stories and impressive experience he creates the butterfly effect.",
+        "role": "Project Engineer"
+      },
+      "ammar": {
+        "name": "Ammar Sammour",
+        "bio": "Ammar is the Coding Instructor at Front-end Web Development Bootcamp in Turkey provided by Re:Coded! He spends his time coaching our fellows to become web development ninjas!",
+        "role": "Lead Engineer"
+      },
+      "ibrahim": {
+        "name": "Ibrahim Aljarad",
+        "bio": "Hey, I'm Ibrahim from Syria, I studied coding during the frontend web developement bootcamp provided by Re:Coded where Flatiron School's curriculum was taught, I also studied economics at Alfurat university. My other experiences include working with several channels and agencies as a satellite producer providing the technical support for the field producing teams.",
+        "role": "Junior Developer"
+      },
+      "majd": {
+        "name": "Majd Ajroudi",
+        "bio": "A junior developer with versatile skills, experience in several engineering areas and keen interest in using tech to solve social issues.",
+        "role": "Junior Developer"
+      },
+      "sule": {
+        "name": "Sule Sinikoglu",
+        "bio": "A Web Developer who writes `good code` for creating positive impact on Global Challenges.",
+        "role": "Junior Developer"
+      },
+      "tarkan": {
+        "name": "Tarkan Mansuroglu",
+        "bio": "A `Re:Coded Front-End Web Development Bootcamp` trained and partially self-taught front-end web developer. He is also a passionate learner, PC enthusiast, gamer and fitness enthusiast. He loves solving real life problems to test and develop his skills. Follow him through social media!",
```

Comment:
backslash is this one. we call this escaping
`\"`
when you want to put a double quote in a double quote or a single quote in a single quote, in almost all languages this is how you do it

Reply:
PTAL